components/grid/overview.md

```diff
@@ -161,6 +161,7 @@ The following table lists Grid parameters, which are not discussed elsewhere in
 
 | Parameter | Type and Default&nbsp;Value | Description |
 | --- | --- | --- |
+| `AdaptiveMode` | `AdaptiveMode` enum <br /> (`None`) | Defines the adaptive mode of the Grid. When set to `Auto`, and the window width is below `768px` or `RootComponentAdaptiveSettings.Medium`, components with popups will render them as an `ActionSheet`. |
```

Comment:
```suggestion
| `AdaptiveMode` | `AdaptiveMode` enum <br /> (`None`) | Defines the adaptive mode of the Grid. When set to `Auto`, and the window width is below [`768px` or `RootComponentAdaptiveSettings.Medium`](slug:adaptive-rendering#rendering-specifics), the Grid will render ins inner popups (for example, FilterMenu, ContextMenu and more) as an `ActionSheet`. |
```
"components with popups" sounds a bit vague to me and it is not clear which components we are referring to. I suggest be more specific.